package.json

```diff
@@ -39,11 +39,20 @@
     "hashlru": "^2.3.0"
   },
   "devDependencies": {
-    "@types/node": "^16.0.0",
+    "@types/node": "^16.11.10",
+    "@typescript-eslint/eslint-plugin": "^5.4.0",
+    "@typescript-eslint/parser": "^5.4.0",
     "art-template": "^4.13.2",
     "cross-env": "^7.0.2",
     "dot": "^1.1.3",
     "ejs": "^3.1.2",
+    "eslint": "^7.32.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-import-resolver-node": "^0.3.6",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^5.1.1",
+    "eslint-plugin-standard": "^5.0.0",
```

Comment:
installing `standard` only, all these dep are not necessary (it ease the dependabot PRs etc..)

Reply:
You lose all the IDE help though, don't you?
Also, this is 1:1 what fastify installs.
Happy to remove it but I wonder why this shouldn't be consistent with fastify.

Reply:
> You lose all the IDE help though, don't you?

It did not happen to me 😕 

> Happy to remove it but I wonder why this shouldn't be consistent with fastify.

I must say that I was not aware of those deps 😱 
I would like to check, but it is fine - it is just a matter of preferences